AcceptanceEKSJenkinsfile

```diff
@@ -68,8 +68,6 @@ pipeline {
                     chmod a+x /usr/local/bin/yq
                     yq --version
 
-                    export ARTIFACTORY_READER=$FUNCTIONAL_USER_USR
-                    export ARTIFACTORY_READER_PW=$FUNCTIONAL_USER_PSW
```

Comment:
Is FUNCTIONAL_USER in line number 43 used somewhere? or remove it.

Reply:
It's used below in line 71-72 to setup SAML_IDP_USER and SAML_IDP_USER_PW